<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminLoginDialog</class>
 <widget class="QDialog" name="AdminLoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>관리자 로그인</string>
  </property>
  <!-- 전체 배경 / 폰트 꾸미기 -->
  <property name="styleSheet">
   <string notr="true">
   QDialog {
       background-color: #fff8e6;       /* 연한 크림색 배경 */
       font-family: "Malgun Gothic";
       font-size: 14px;
   }
   QLabel#title_label {
       font-size: 22px;
       font-weight: bold;
   }
   QLineEdit {
       border: 1px solid #cccccc;
       border-radius: 6px;
       padding: 6px 8px;
       background-color: #ffffff;
   }
   QPushButton {
       background-color: #ffb74d;
       border-radius: 8px;
       padding: 6px 14px;
       font-weight: bold;
       color: #333333;
   }
   QPushButton:hover {
       background-color: #ffa726;
   }
   QPushButton#cancel_btn {
       background-color: #e0e0e0;
   }
   QPushButton#cancel_btn:hover {
       background-color: #d5d5d5;
   }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="contentsMargins">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <widget class="QLabel" name="title_label">
     <property name="text">
      <string>관리자 로그인</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>12</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="id_label">
       <property name="text">
        <string>아이디</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="id_edit">
       <!-- ⭐ placeholderText : 회색 안내 문구 (입력하면 자동으로 사라짐) -->
       <property name="placeholderText">
        <string>아이디를 입력해주세요</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="pw_label">
       <property name="text">
        <string>비밀번호</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pw_edit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <!-- ⭐ 비밀번호 placeholder -->
       <property name="placeholderText">
        <string>비밀번호를 입력해주세요</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="login_btn">
       <property name="text">
        <string>로그인</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="text">
        <string>취소</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
